src/scripts.js

```diff
@@ -132,79 +121,87 @@ function updateStar(event) {
 
 function cardButtonConditionals(event) {
   if (event.target.classList.contains('favorite')) {
-    favoriteCard(event)
+    favoriteCard(event, cookbook)
   } else if (event.target.classList.contains("card-picture")) {
     displayDirections(event)
-  } else if (event.target.classList.contains("add-button")) {
-    let recipe = cookbook.recipes.find((recipe) => {
-      if (recipe.id === Number(event.target.id)) {
-        return recipe
-      }
-    })
-    user.addToRecipeArray(recipe, user.recipesToCook)
+    // } else if (event.target.classList.contains("add-button")) {
```

Comment:
Commented out code

Reply:
I was trying to handle all the displayDirections.

Reply:
We are using this code later if we get to displaying the added recipes to cook in the array attached to the user.